src/test/resources/pages/InstructorCourseEnrollPageResult.html

```diff
@@ -1,162 +1,238 @@
-<div class="container" id="frameBodyWrapper">
-        <div id="headerOperation">
-            <h1>Enrollment Results for CCEnrollUiT.CS2104</h1>
-        </div>
-
-        <div class="alert alert-success">
-        <form name="goBack" action="/page/instructorCourseEnrollPage?courseid=CCEnrollUiT.CS2104&amp;user=CCEnrollUiT.teammates.test" method="post" role="form"> 
-            Enrollment Successful. Summary given below. Click <a id="edit_enroll" href="javascript:document.forms['goBack'].submit()">here</a> to do further changes to the student list.
-            
-        <input name="courseid" value="CCEnrollUiT.CS2104" type="hidden">
-        <input name="enrollstudents" value="Section | Team | Name | Email | Comments
+   <div class="container" id="frameBodyWrapper">
```

Comment:
Did the browser tests fail when you ran the tests? If they did not fail, then it is better to leave the expected html files as they are. I think we should stop modifying the html files if there's no need to, and only modify the html using godmode.